### DISCUSSION TITLE
How to enable Github Copilot on lazyvim ?

### DISCUSSION THREAD

**Discussion by MirceaDumitrescu** [LazyVim/LazyVim#401] [Q&A] [answered]
Hey guys, started using lazyvim a few days ago but i am hesitant since i was not able to make github copilot work on it.

Checking with :Copilot status , everything looks ok, but no suggestions are showing up in the code.

Anyone else has experienced this before?

**Comment by folke** [comment, marked as answer]
Did you use the copilot extra?

Did you run `:Copilot auth`?

**Reply by MirceaDumitrescu** [reply]
Not sure what you mean by copilot extra?

I ran :Copilot setup and :Copilot enable. Everything worked fine.

**Reply by folke** [reply]
https://www.lazyvim.org/plugins/extras/coding.copilot

**Reply by folke** [reply]
You obviously dont need to use that extra. But then you need to configure it yourself properly.
The extra will add copilot suggestions to cmp

**Reply by MirceaDumitrescu** [reply]
It seems i was implement the plugin wrong all together. I dont know how i missed that setting. Thank you so much
